react/Shelf.js

```diff
@@ -25,11 +25,12 @@ const { ProductListProvider } = ProductListContext
 /**
  * Shelf Component. Queries a list of products and shows them.
  */
-const Shelf = ({
-  data,
-  productList = ProductList.defaultProps,
-  paginationDotsVisibility = 'visible',
-}) => {
+const Shelf = props => {
+  const {
+    data,
+    paginationDotsVisibility = 'visible',
+    productList = ProductList.defaultProps,
+  } = props
```

Comment:
Why change this?

Reply:
Oh, I did it just because first I was getting all the new props of `ProductList` here and I think is easier to debug (aka `console.log(props)`) this way, but then I pass all the props to `ProductList` and didn't change this, since this doesn't affect anything in the code